src/components/BulkOperations/index.jsx

```diff
@@ -0,0 +1,194 @@
+import  { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import Papa from 'papaparse';
+
+import { SelectMenu, MenuItem, Dropzone, DataTable, Alert} from '@openedx/paragon';
+import Collapsible from '../Collapsible'
+import LoadingSpinner from '../LoadingSpinner';
+import { formatDate } from '../../utils';
+import DiscoveryDataApiService from '../../data/services/DiscoveryDataApiService';
+
+import './BulkOperations.scss';
+
+const availableOperations = {
+    "course_create": "Bulk Create",
+    "course_partial_update": "Bulk Course Update",
+    "course_run_partial_update": "Bulk CourseRun Update",
+    "course_rerun": "Bulk Rerun"
+}
+
+
+function BulkOperations(){
+    const [bulkOperationId, setBulkOperationId] = useState(null);
+    const [fileContent, setFileContent] = useState(null);
+    const [file, setFile] = useState(null);
+    const [isLoading, setIsLoading] = useState(true);
+    const [submitSuccess, setSubmitSuccess] = useState(null);
+    const [historicalRecords, setHistoricalRecords] = useState(null);
+
+    function filteredHistoricalRecords() {
+        if (historicalRecords == null){
+            return []
+        }
+        return historicalRecords.filter(rec => !bulkOperationId || rec.task_type === bulkOperationId)
+    }
+
+    function handleUploadNew(){
+        setFileContent(null);
+        setFile(null);
+    }
+
+    function handleFileUpload({fileData}){
+        let file = fileData.get('file');
+        file.text().then(content => {setFile(file); setFileContent(content);});
+    }
```

Comment:
Same here, can we convert all of them arrow functions

Reply:
Not sure what benefits that'd offer.